Gemfile.lock

```diff
@@ -2,78 +2,263 @@ PATH
   remote: .
   specs:
     decidim-internal_evaluation (0.28.1)
-      decidim-core (>= 0.28.1)
+      decidim-core (>= 0.28)
+      decidim-proposals (>= 0.28)
+      decidim-templates (>= 0.28)
+      deface (~> 1.9)
+
+PATH
+  remote: /home/edu/proyectos/decidim/decidim_version_movil/decidim
```

Comment:
I was trying to create the `test_app` to run the tests locally and I realized about this. Could you change it to a version not coming from your local setup?

Reply:
Ah, the problem is that the module requires Decidim 0.29 release. I'm going to try to update the dependencies with the recent 0.29.0.rc1

Reply:
You could even use 0.29.0.rc4, which seems to be the last release candidate. We could even wait for the official 0.29.0 version which will be released in a few days.

Reply:
Done! I think that the tests will run correctly in your local environment